components/LangButton.tsx

```diff
@@ -5,6 +5,10 @@ import { ChevronDown } from 'react-feather'
 
 type Langs = { [key: string]: { name: string; icon: JSX.Element } }
 const langs: Langs = {
+  de: {
+    name: 'Deutsch',
+    icon: <img src="https://twemoji.maxcdn.com/svg/1f1e9-1f1ea.svg" className="w-5 h-5" />
+  },
```

Comment:
This should be at the end of the `langs` object, so it is at the end of the dropdown

Reply:
Why? First comes German, then English, then the other languages! Looks otherwise weird

Reply:
And I'm not saying this because I'm German

Reply:
I want to prioritize English first, then French, then German - that's the order of priority I want to display.

Reply:
Fix = bc80e35

Reply:
Fixed = bc80e35